docs/CHANGELOG.md

```diff
@@ -1,15 +1,17 @@
 Version 3.30.1 (March 1, 2023)
---------------------------------
+------------------------------
 
 **User-visible changes:**
 
 **Implementation details:**
 
+Changed uses of `Set<AnnotationMirror>` to `AnnotationMirrorSet` including in APIs.
```

Comment:
You also deleted some methods in AnnotationUtils and moved AnnotationMirrorSet and AnnotatinMirrorMap.